extension/copy-gist.js

```diff
@@ -0,0 +1,24 @@
+/* globals utils */
+
+'use strict';
+
+window.addGistCopyButton = () => {
+	// Button already added (partial page nav), or non-text file
+	if ($('.copy-btn').length > 0) {
+		return;
+	}
+	const $gistsSibling = $('.file-actions > .btn.btn-sm');
+
+	for (const anchor of $gistsSibling) {
+		const gistUri = anchor.href;
+		if (gistUri.split('.').pop().toLowerCase() !== 'md') {
```

Comment:
Didn't get this :slightly_frowning_face: Idk the file extensions which don't have a source code view... I think u mean that there is no numbering on the RHS like in the above `.js` file. Right ??